cclib/parser/nwchemparser.py

```diff
@@ -59,6 +59,13 @@ def extract(self, inputfile, line):
                         "package_version"
                     ] = f"{self.metadata['package_version']}+{revision}"
 
+        if "nproc" in line:
+            self.metadata['num_processors'] = line.split()[-1]
+        if "Memory information" in line:
+            self.skip_lines(inputfile,['d','b','heap','stack','global'])
+            self.metadata['memory'] = line.split()[-2:]

```

Comment:
`int`, but we also want this in bytes.  I don't know if `Mbytes` is actually megabytes.  I guess it isn't, since the number of doubles should be the number of bytes.